src/pages/index.tsx

```diff
@@ -0,0 +1,56 @@
+import type { NextPage } from "next";
+import Head from "next/head";
+
+import { Hero } from "../components/Hero/Hero";
+import MemberWall from "../components/MemberWall";
+import Navigation from "../components/Navigation";
+import UpcomingEvents from '../components/UpcomingEvents'
+import styles from "../styles/Home.module.css";
+
+interface Props {
+  members: any;
+}
+
+// lint-ignore:next-line Property 'members' does not exist on type
+const Home: NextPage<Props> = ({ members }) => {
```

Comment:
As much as possible we should try to avoid `lint-ignore`.

Reply:
you can delete that line before you merge, or i can delete it in an update. i just needed that for dev, it was fixed on build